policyengine_us/variables/gov/irs/gross_income/disqualified_income.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class earned_income_disqualified_income(Variable):
```

Comment:
is it possible to avoid using "income" twice in the same variable name.. more like "disqualified_earned_income"